control/state.py

```diff
@@ -108,7 +180,7 @@ def __init__(self, config):
         conn = rados.Rados(conffile=ceph_conf)
         conn.connect()
         self.ioctx = conn.open_ioctx(ceph_pool)
```

Comment:
I assume any of the 3 lines above can raise an exception so why not move it into the try block so we get some logging if it happens